src/components/Filter/index.js

```diff
@@ -61,30 +62,32 @@ export default class Filter extends Component {
   };
 
   render() {
-    const { filterRenderer, data, dataKey, filterIconRenderer } = this.props;
+    const { filterRenderer, data, dataKey, filterTrigger } = this.props;
     return (
-      <div
-        className="Sticky-React-Table--Header-Column-Filter"
-        style={filtersStyles}
-      >
-        <div
-          className="Sticky-React-Table--Header-Column-Filter-Icon"
-          onClick={this.handleIconClick}
-          ref={this.handleIconRef}
-          style={filterIconStyles}
+      <Fragment>
+        <span
```

Comment:
Instead of rendering inside a `span`, render inside a `Fragment` and let the icon be a `span` and the dropdown be a `div`.

Reply:
I have made the icon as span and dropdown as div. I need to pass style to the parent. So i can't use `Fragment`